zerver/views/home.py

```diff
@@ -271,7 +264,7 @@ def home_real(request):
         notifications_stream  = notifications_stream,
         cross_realm_bots      = list(get_cross_realm_dicts()),
         unread_count          = approximate_unread_count(user_profile),
-        furthest_read_time    = sent_time_in_epoch_seconds(latest_read),
+        furthest_read_time    = sent_time_in_epoch_seconds(latest_visit),
```

Comment:
Lets call this `last_presence_time`, and also get rid of the `sent_time_in_epoch_seconds` (and just send the datetime or None).
I think this is the only place `sent_time_in_epoch_seconds` is used, so you can delete the function entirely.